DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -64,11 +64,9 @@ mixin block-licenseAndCopyright
 		.copyright(data-book="copyright", lang="*").Credits-Page-style
 			{copyright}
 		.licenseBlock
-			img.licenseImage(src="" data-book="licenseImage")
-			.licenseUrl(data-book="licenseUrl", lang="en")
-				{License Url}
-			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style
-				{License Description}
+			img.licenseImage(src="license.png" data-book="licenseImage")
+			.licenseUrl(data-book="licenseUrl", lang="en") http://creativecommons.org/licenses/by/4.0/
+			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style You are free to make commercial use of this work. You may adapt and add to this work. You must keep the copyright and credits for authors, illustrators, etc.
```

Comment:
I think what we have is good enough. If someone complains we can add another issue as needed. The main thing is that new books will get the right license in the right language.